### PR TITLE
dotnet-build: add German translation

### PR DESCRIPTION
**Pull request by oliver-brandt**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.de/common/dotnet-build.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# dotnet build
+
+> Baut eine .NET Anwendung und ihre Abhängigkeiten.
+> Mehr Informationen: <https://learn.microsoft.com/de-de/dotnet/core/tools/dotnet-build>.
```
Comment (kbdharun) [inline pages.de/common/dotnet-build.md:4]:
```suggestion
> Weitere Informationen: <https://learn.microsoft.com/de-de/dotnet/core/tools/dotnet-build>.
```
[Reference](https://github.com/tldr-pages/tldr/blob/main/scripts/set-more-info-link.py)

**Review by kbdharun — COMMENTED** [review]
Can you translate command arguments like paths,etc inside `{{}}` too?

Skip command arguments in line 24 alone.

File: pages.de/common/dotnet-build.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+`dotnet build --verbosity {{quiet|minimal|normal|detailed|diagnostic}}`
+
+- Kompiliert für eine spezifische Laufzeitumgebung:
+
+`dotnet build --runtime {{Laufzeitbezeichner}}`
+
+- Kompiliert in ein spezifischen Ordner:
+
+`dotnet build --output {{Pfad/zum/Verzeichnis}}`
```
Comment (kbdharun) [inline pages.de/common/dotnet-build.md:32]:
```suggestion
`dotnet build --output {{pfad/zu/verzeichnis}}`
```

File: pages.de/common/dotnet-build.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+> Baut eine .NET Anwendung und ihre Abhängigkeiten.
+> Weitere Informationen: <https://learn.microsoft.com/de-de/dotnet/core/tools/dotnet-build>.
+
+- Kompiliert das Projekt oder eine Projektmappendatei in dem aktuellen Ordner:
+
+`dotnet build`
+
+- Kompiliert ein .NET Projekt oder eine Projektmappendatei Debug-Modus:
+
+`dotnet build {{Pfad/zum/Projekt_oder_Projektmappendatei}}`
```
Comment (kbdharun) [inline pages.de/common/dotnet-build.md:12]:
```suggestion
`dotnet build {{pfad/zu/projekt_oder_projektmappendatei}}`
```

File: pages.de/common/dotnet-build.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
+# dotnet build
+
+> Baut eine .NET Anwendung und ihre Abhängigkeiten.
```
Comment (marchersimon) [inline pages.de/common/dotnet-build.md:3]: Descriptions should always be in imperative form.
```suggestion
> Baue .NET Anwendungen und ihre Abhängigkeiten.
```

File: pages.de/common/dotnet-build.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+> Weitere Informationen: <https://learn.microsoft.com/de-de/dotnet/core/tools/dotnet-build>.
+
+- Kompiliert das Projekt oder eine Projektmappendatei in dem aktuellen Ordner:
```
Comment (marchersimon) [inline pages.de/common/dotnet-build.md:6]:
```suggestion
- Kompiliere das Projekt oder eine Projektmappendatei in dem aktuellen Verzeichnis:
```

File: pages.de/common/dotnet-build.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
+# dotnet build
+
+> Baut eine .NET Anwendung und ihre Abhängigkeiten.
+> Weitere Informationen: <https://learn.microsoft.com/de-de/dotnet/core/tools/dotnet-build>.
```
Comment (marchersimon) [inline pages.de/common/dotnet-build.md:4]: We try to keep the links the same in every translation. If you have the correct locale set, you will be forwarded automatically anyways.
```suggestion
> Weitere Informationen: <https://learn.microsoft.com/dotnet/core/tools/dotnet-build>.
```

File: pages.de/common/dotnet-build.md [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+
+- Kompiliert das Projekt oder eine Projektmappendatei in dem aktuellen Ordner:
+
+`dotnet build`
+
+- Kompiliert ein .NET Projekt oder eine Projektmappendatei Debug-Modus:
```
Comment (marchersimon) [inline pages.de/common/dotnet-build.md:10]:
```suggestion
- Kompiliere ein .NET Projekt oder eine Projektmappendatei im Debug-Modus:
```

File: pages.de/common/dotnet-build.md [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+
+`dotnet build {{Pfad/zum/Projekt_oder_Projektmappendatei}}`
+
+- Kompiliert im Release-Modus:
```
Comment (marchersimon) [inline pages.de/common/dotnet-build.md:14]:
```suggestion
- Kompiliere im Release-Modus:
```

File: pages.de/common/dotnet-build.md [hunk 9]
```diff
@@ -0,0 +1,32 @@
...
+
+`dotnet build --configuration {{Release}}`
```
Comment (marchersimon) [inline pages.de/common/dotnet-build.md:16]: The command arguments should always be lowercase
```suggestion
`dotnet build --configuration {{release}}`
```

File: pages.de/common/dotnet-build.md [hunk 10]
```diff
@@ -0,0 +1,32 @@
...
+
+- Kompiliert ohne die Abhängigkeiten wiederherzustellen:
+
```
Comment (marchersimon) [inline pages.de/common/dotnet-build.md:19]:
```suggestion
- Kompiliere ohne die Abhängigkeiten wiederherzustellen:

```

File: pages.de/common/dotnet-build.md [hunk 11]
```diff
@@ -0,0 +1,32 @@
...
+`dotnet build --no-restore`
+
+- Kompiliert mit einem spezifische Ausführlichkeitsgrad:
```
Comment (marchersimon) [inline pages.de/common/dotnet-build.md:22]:
```suggestion
- Kompiliere mit einem bestimmten Ausführlichkeitsgrad:
```

File: pages.de/common/dotnet-build.md [hunk 12]
```diff
@@ -0,0 +1,32 @@
...
+
+`dotnet build --verbosity {{quiet|minimal|normal|detailed|diagnostic}}`
+
+- Kompiliert für eine spezifische Laufzeitumgebung:
```
Comment (marchersimon) [inline pages.de/common/dotnet-build.md:26]:
```suggestion
- Kompiliere für eine bestimmte Laufzeitumgebung:
```

File: pages.de/common/dotnet-build.md [hunk 13]
```diff
@@ -0,0 +1,32 @@
...
+
+`dotnet build --runtime {{Laufzeitbezeichner}}`
```
Comment (marchersimon) [inline pages.de/common/dotnet-build.md:28]:
```suggestion
`dotnet build --runtime {{laufzeitbezeichner}}`
```

File: pages.de/common/dotnet-build.md [hunk 14]
```diff
@@ -0,0 +1,32 @@
...
+
+- Kompiliert in ein spezifischen Ordner:
```
Comment (marchersimon) [inline pages.de/common/dotnet-build.md:30]:
```suggestion
- Kompiliere in einen bestimmten Ordner:
```

**Comment by marchersimon** [comment]
@oliver-brandt could you please apply the suggestions? If you need help with git or GitHub, just tell us.

File: pages.de/common/dotnet-build.md (hunk 6 above)
Comment (kbdharun) [reply pages.de/common/dotnet-build.md:4]: Yes, based on your system locale the links automatically get redirected to translated versions. That's why we don't use l10n Microsoft URLs and instead use generalised ones.

**Comment by oliver-brandt** [comment]
We have different views on spelling.

**Comment by marchersimon** [comment]
@oliver-brandt what exactly do you mean? There's still room for discussion, for some things (like imperative mood) have to be respected.

**Pull request closed without merging**